core/src/test/java/nu/yona/server/goals/service/ActivityCategoryServiceTest.java

```diff
@@ -0,0 +1,121 @@
+package nu.yona.server.goals.service;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.collection.IsIterableContainingInAnyOrder.containsInAnyOrder;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.mockito.stubbing.Answer;
+
+import nu.yona.server.goals.entities.ActivityCategory;
+import nu.yona.server.goals.entities.ActivityCategoryRepository;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ActivityCategoryServiceTest
+{
+	private Set<ActivityCategory> activityCategories = new HashSet<ActivityCategory>();
+
+	@Mock
+	ActivityCategoryRepository mockRepository = mock(ActivityCategoryRepository.class);
+	@InjectMocks
+	private ActivityCategoryService service = new ActivityCategoryService();
+
+	private ActivityCategory gambling;
+	private ActivityCategory news;
+
+	@Before
+	public void setUp()
```

Comment:
This setup is a straight copy of  the one in ``ActivityCategoryServiceIntegrationTest``. Can't you put it somewhere where it can be shared? Common base class?